### ISSUE TITLE
[Bug] baichuan-7b AWQ quantize key error

### ISSUE THREAD

**Issue by 77h2l** [InternLM/lmdeploy#573]
### Checklist

- [ ] 1. I have searched related issues but cannot get the expected help.
- [ ] 2. The bug has not been fixed in the latest version.

### Describe the bug

Traceback (most recent call last):
File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
return _run_code(code, main_globals, None,
File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
exec(code, run_globals)
File "/opt/tritonserver/lmdeploy/lmdeploy/lite/apis/auto_awq.py", line 85, in
fire.Fire(auto_awq)
File "/usr/local/lib/python3.8/dist-packages/fire/core.py", line 141, in Fire
component_trace = _Fire(component, args, parsed_flag_args, context, name)
File "/usr/local/lib/python3.8/dist-packages/fire/core.py", line 475, in _Fire
component, remaining_args = _CallAndUpdateTrace(
File "/usr/local/lib/python3.8/dist-packages/fire/core.py", line 691, in _CallAndUpdateTrace
component = fn(*varargs, **kwargs)
File "/opt/tritonserver/lmdeploy/lmdeploy/lite/apis/auto_awq.py", line 52, in auto_awq
fc2fcs = FC_FCS_MAP[layer_type]
KeyError: 'DecoderLayer'

### Reproduction

use the following command "python3 -m lmdeploy.lite.apis.auto_awq --model /model/Baichuan-7B/ --w_bits 4 --w_group_size 128 --work_dir ./work_dir/"

returns

Traceback (most recent call last):
File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
return _run_code(code, main_globals, None,
File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
exec(code, run_globals)
File "/opt/tritonserver/lmdeploy/lmdeploy/lite/apis/auto_awq.py", line 85, in
fire.Fire(auto_awq)
File "/usr/local/lib/python3.8/dist-packages/fire/core.py", line 141, in Fire
component_trace = _Fire(component, args, parsed_flag_args, context, name)
File "/usr/local/lib/python3.8/dist-packages/fire/core.py", line 475, in _Fire
component, remaining_args = _CallAndUpdateTrace(
File "/usr/local/lib/python3.8/dist-packages/fire/core.py", line 691, in _CallAndUpdateTrace
component = fn(*varargs, **kwargs)
File "/opt/tritonserver/lmdeploy/lmdeploy/lite/apis/auto_awq.py", line 52, in auto_awq
fc2fcs = FC_FCS_MAP[layer_type]
KeyError: 'DecoderLayer'

it seems problems occurs from here https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/lite/quantization/awq.py#L24

also qwen-7b after awq quantized, when convert to turbomind, will occur tokenizer.model not found

could you please affirm, thx

### Error traceback

```Shell
it seems problems occurs from here https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/lite/quantization/awq.py#L24
```

**Comment by 77h2l** [comment]
hello, is there any response?

**Comment by pppppM** [comment]
@77h2l  This bug has been fixed(#605 ) and the latest main branch can be used.
